GitUI/CommandsDialogs/FormPull.cs

```diff
@@ -1163,6 +1164,17 @@ private void PruneTags_CheckedChanged(object sender, EventArgs e)
             AllTags.Checked = AllTags.Checked || PruneTags.Checked;
         }
 
+        private void DisableMergeOptionsWhenCurrentBranchIsLocal()
+        {
+            var isLocalBranch = string.IsNullOrEmpty(Module.GetRemoteBranch(localBranch.Text));
+
+            Merge.Checked = !isLocalBranch;
+            Merge.Enabled = !isLocalBranch;
+            Rebase.Enabled = !isLocalBranch;
+            Fetch.Checked = !isLocalBranch;
+            Fetch.Enabled = !isLocalBranch;
```

Comment:
if (!isLocalBranch)
{
  "Do not merge".Checked = true;
}

Reply:
What you mean by this?